src/web/app/components/question-types/question-statistics/rubric-question-statistics.component.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="col-sm-8" style="text-align: right;">
       <label class="form-check-label">
-        <input id="exclude-self-checkbox" type="checkbox" [(ngModel)]="excludeSelf" (ngModelChange)="getTableData()" class="form-check-input"> <span container="body" class="ngb-tooltip-class" ngbTooltip="Excludes giver's responses to himself/herself from statistics">Exclude self evaluation</span>
+        <input id="exclude-self-checkbox" type="checkbox" [(ngModel)]="excludeSelf" (ngModelChange)="getTableData()" class="form-check-input"> <span container="body" class="ngb-tooltip-class" ngbTooltip="Excludes giver's responses to himself/herself from statistics" style="margin-left: .33rem;">Exclude self evaluation</span>
```

Comment:
Quick note: let's try to use [bootstrap styles](https://getbootstrap.com/docs/5.0/utilities/spacing/#margin-and-padding) as much as possible. But let's see if there are any other checkbox spaces issues first!